manifest-pcf.yml

```diff
@@ -8,16 +8,17 @@ applications:
   - fortunes-db
   - config-server
   - service-registry
-  #env:
-    # Replace with API URI of target PCF environment
-    #CF_TARGET: https://api.yourpcfenvironment.local
+  env:
+    CF_TARGET: https://api.cf.gold.springapps.io
+    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{"enabled":false}'
```

Comment:
Why is this environment variable needed now?

Reply:
The java buildpack adds spring-cloud-connectors during autoreconfiguration, which clashes with java-cfenv. I need to look into what can be done with the jbp in this regard as connectors are depracated.

Reply:
Is this documented somewhere? Seems like an important thing to be remembered of that should be the default.

Reply:
Right now the java-cfenv library will fail on application start if it finds connectors on the classpath. It gives an error message that includes instructions to add this very environment variable. :)